### PR TITLE
January Release Notes: Addition of Event Hubs information

### PR DESCRIPTION
**Pull request by jsquire**
Update of release notes to include Event Hubs.

### CODE REVIEW

File: releases/2020-01/dotnet.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
-title: Azure SDK for .NET (January 2020)
-layout: post
-date: January 2020
+title: Azure SDK for .NET (Latest)
```
Comment (adrianhall) [inline releases/2020-01/dotnet.md]: Please revert this change - the title needs the month/year in it.

File: releases/2020-01/dotnet.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
...
+layout: default
```
Comment (adrianhall) [inline releases/2020-01/dotnet.md]: Wrong layout - also the date needs to be added.

File: releases/2020-01/dotnet.md [hunk 3]
```diff
@@ -51,9 +52,15 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
...
 
+### Event Hubs
+
+- Namespaces have been reorganized to align types to their functional area. Cross-functional types have been left in the root namespace while specialized types were moved to the Producer, Consumer, or Processor namespaces.
+
+- The hierarchy of custom exceptions has been flattened, with only the EventHubsException remaining. The well-known failure scenarios that had previously been represented as stand-alone types are now exposed by the `Reason` property.  Please see the [Event Hubs README](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/eventhub/Azure.Messaging.EventHubs#event-hubs-exception) for details.
+
 ## Quick Links
 
-{% assign packages = site.data.releases.latest.dotnet-packages %}
```
Comment (adrianhall) [inline releases/2020-01/dotnet.md]: Please go with what Wes advises on this - I think we haven't decided on the month version of the packages yet.

File: releases/2020-01/dotnet.md (hunk 1 above)
Comment (jsquire) [reply releases/2020-01/dotnet.md]: Gah.  Sorry, looks like my rebase did some ... interesting... things after yesterday's churn.   Sorry about that, I'll get it fixed up.

File: releases/2020-01/dotnet.md (hunk 2 above)
Comment (heaths) [reply releases/2020-01/dotnet.md]: Basically, just undo any changes to the front matter. It was all already correct after my merge.

File: releases/2020-01/dotnet.md [hunk 4]
```diff
@@ -51,9 +52,15 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
 
 - Challenge-based authentication requests are only sent over HTTPS.
 
+### Event Hubs
```
Comment (heaths) [inline releases/2020-01/dotnet.md]: I'd put this before Key Vault. I was trying to keep the sections alphabetized. Prior, it was all just random order. We can do better for customers.

File: releases/2020-01/dotnet.md (hunk 3 above)
Comment (heaths) [reply releases/2020-01/dotnet.md]: Yes, this needs to be just "latest". That 202001 directory shouldn't even exist under _data now.

**Comment by jsquire** [comment]
The rebase issues should now be resolved and the unintentional changes restored.   Took Heath's suggestion and alphabetized the product names while I was there.

File: releases/2020-01/dotnet.md (hunk 3 above)
Comment (jsquire) [reply releases/2020-01/dotnet.md]: Agreed.  The change was unintended;  fixed the rebase issues and these were reverted.
